app/package.json

```diff
@@ -58,109 +58,109 @@
     "@formatjs/intl-numberformat": "7.2.4",
     "@formatjs/intl-pluralrules": "4.1.4",
     "@formatjs/intl-relativetimeformat": "9.3.1",
-    "@hyperledger/anoncreds-react-native": "^0.1.0",
-    "@hyperledger/aries-askar-react-native": "^0.1.1",
-    "@hyperledger/aries-bifold-core": "1.0.0-alpha.150",
-    "@hyperledger/aries-bifold-verifier": "1.0.0-alpha.150",
-    "@hyperledger/aries-oca": "1.0.0-alpha.150",
-    "@hyperledger/aries-react-native-attestation": "1.0.0-alpha.150",
-    "@hyperledger/indy-vdr-react-native": "^0.1.0",
+    "@hyperledger/anoncreds-react-native": "0.1.0",
+    "@hyperledger/aries-askar-react-native": "0.1.1",
+    "@hyperledger/aries-bifold-core": "1.0.0-alpha.155",
+    "@hyperledger/aries-bifold-verifier": "1.0.0-alpha.155",
+    "@hyperledger/aries-oca": "1.0.0-alpha.155",
+    "@hyperledger/aries-react-native-attestation": "1.0.0-alpha.155",
+    "@hyperledger/indy-vdr-react-native": "0.1.0",
```

Comment:
maybe changing this back to `^0.1.0` would fix the issue?

Reply:
instead I removed the caret from the resolution block in the top level package.json and that worked